qcodes/tests/dataset/test_plotting.py

```diff
@@ -148,3 +149,97 @@ def test_appropriate_kwargs():
     with _appropriate_kwargs('2D_point', False, **{}) as ap_kwargs:
         assert len(ap_kwargs) == 1
         assert ap_kwargs['cmap'] == qc.config.plotting.default_color_map
+
+
+def test__complex_to_real_preparser():
+
+    # FIRST CASE: one real setpoint, one complex value
```

Comment:
Shall the `test__complex_to_real_preparser` be split into at least two independent ones (ideally into 4 parts actually)? I'd also suggest for this test to go into its own file.

Reply:
I agree with splitting it into two tests. I don't think it should go into a separate file unless the preparser also leaves the plotting module (into `plotting_helpers.py` or smth)